src/components/OperatorChecklist.tsx

```diff
@@ -0,0 +1,289 @@
+import React, { ComponentProps, ReactNode, useEffect, useState } from 'react'
+import styled, { css } from 'styled-components'
+import CheckCircleIcon from '@atlaskit/icon/glyph/check-circle'
+import JiraFailedBuildStatusIcon from '@atlaskit/icon/glyph/jira/failed-build-status'
+import { defaultChainConfig } from '~/getters/getChainConfig'
+import { useOperatorByIdQuery } from '~/hooks/operators'
+import useOperatorLiveNodes from '~/hooks/useOperatorLiveNodes'
+import { toAtto } from '~/marketplace/utils/math'
+import { getNativeTokenBalance } from '~/marketplace/utils/web3'
+import { toBN } from '~/utils/bn'
+import Spinner from '~/shared/components/Spinner'
+import SvgIcon from '~/shared/components/SvgIcon'
+import { Separator } from '~/components/Separator'
+import { TABLET } from '~/shared/utils/styled'
+import { StatCellLabelTip } from '~/components/StatGrid'
+import { useInterceptHeartbeats } from '~/hooks/useInterceptHeartbeats'
+import { SponsorshipPaymentTokenName } from './SponsorshipPaymentTokenName'
+
+export function OperatorChecklist({ operatorId }: { operatorId: string | undefined }) {
+    const { funded, nodesDeclared, nodesFunded, nodesReachable, nodesRunning } =
+        useOperatorChecklist(operatorId)
+
+    return (
+        <>
+            <ChecklistItem
+                state={funded}
+                tip={
+                    <p>
+                        You need to deposit <SponsorshipPaymentTokenName /> tokens into
+                        your operator contract before you can stake on sponsorships or
+                        receive delegations.
+                    </p>
+                }
+            >
+                Operator funded
+            </ChecklistItem>
+            <Separator />
+            <ChecklistItem
+                state={nodesDeclared}
+                tip={
+                    <p>
+                        Configure your nodes with private keys and declare the
+                        corresponding addresses to the operator contract.
+                    </p>
+                }
+            >
+                Node addresses declared
+            </ChecklistItem>
+            <Separator />
+            <ChecklistItem
+                state={nodesFunded}
+                tip={
+                    <p>
+                        Your node addresses must be funded with some MATIC for Polygon
+                        transaction fees. This alert triggers if a balance is less than
+                        0.1 MATIC.
```

Comment:
is this value in the global config? if so it would be good to fetch it from there instead of hardcoding